### ISSUE TITLE
[REQUEST]: JUnit or Markdown report from Nightwatch daily accessibility tests on static content

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#21671]
## Issue Description
VSP accessibility specialists would be better equipped to help teams identify axe violations in static content if we had a report in JUnit or Markdown that was derived from the tests' Jenkins logs.

This work is related to @timwright12 work to move static content a11y tests into daily integration tests: https://github.com/department-of-veterans-affairs/vets-website/pull/16402

---
## Tasks
- [ ] Add a JUnit or Markdown report from Jenkins logs on static content a11y checks
- [ ] Link the report in Slack notification

## Acceptance Criteria
- [ ] JUnit or Markdown report from Jenkins logs
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by timwright12** [comment]
Closing out, as this work was completed by QA Standards
